samples/Chat/src/app/utils/utils.ts

```diff
@@ -4,13 +4,9 @@
 import { GUID_FOR_INITIAL_TOPIC_NAME } from './constants';
 
 declare let __BUILDTIME__: string; // Injected by webpack
+declare let __CHATVERSION__: string; // Injected by webpack
+export const chatSDKVersion = __CHATVERSION__;
```

Comment:
Follow the syntax of getBuildTime to make it consistent? Just make it a function and return the value